src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -157,6 +157,22 @@ public static class ConfigurationKeys
         /// </summary>
         public const string ProfilerLogPath = "DD_TRACE_LOG_PATH";
 
+        /// <summary>
+        /// Configuration key for when a standalone instance of a trace agent needs to be started.
+        /// </summary>
+        public const string AgentPath = "DD_AGENT_PATH";
```

Comment:
We may want this to say "Trace Agent" since the Agent is a different executable.
```suggestion
        public const string TraceAgentPath = "DD_TRACE_AGENT_PATH";
```

Reply:
+1